scss/modifierClasses/_list.scss

```diff
@@ -105,10 +104,10 @@ Class                   | Description
 }
 
 .list-item--gutters {
-	@include responsiveVarContext--base() {
-		padding-left: $base;
-		padding-right: $base;
-	}
+	padding-left: $space;
+	padding-right: $space;
+	padding-left: var(--baseSpacing);
+	padding-right: var(--baseSpacing);
```

Comment:
looks like the `$space` valued properties are redundant here - could we use `var(--baseSpacing)` by itself?

Reply:
We could, but it would completely remove padding in [browsers that don't support custom properties](https://caniuse.com/#feat=css-variables)